### DISCUSSION TITLE
Inquiry into system image for MuSHR

### DISCUSSION THREAD

**Discussion by ROMANT21** [prl-mushr/mushr#117] [Q&A] [answered]
Hey all,

I was curious if you all had any advice on how to take a system image of the MuSHR. I've made a customized version that runs ROS Noetic natively on Ubuntu 20, but I haven't had much look cloning the image to install on other cars. 

My main question is, what tool was used to make the system image that MuSHR provides for the Jetson Nano?

**Comment by schmittlema** [comment, marked as answer]
Hi Romant21,
We don't flash now just cause it proved error prone but back in the day we followed a similar procedure to [this](https://docs.nvidia.com/jetson/archives/l4t-archived/l4t-3261/index.html#page/Tegra%20Linux%20Driver%20Package%20Development%20Guide/flashing.html#wwpID0E0MQ0HA).

**Reply by ROMANT21** [reply]
I see, if you don't mind me asking, what is the standard procedure now?

**Reply by schmittlema** [reply]
The standard procedure now is we download the latest jetpack and do the install mainly via docker which is covered in our [quickstart tutorial](https://mushr.io/tutorials/noetic_quickstart/) and [software setup tutorial](https://mushr.io/tutorials/noetic_first_steps/)
